### DISCUSSION TITLE
Alpha configurable using .Xresources

### DISCUSSION THREAD

**Discussion by ghost** [bakkeby/dwm-flexipatch#351] [Ideas]
With all of the alpha patches in all suckless software, there's no way to override the alpha through.. for example the .Xresources patch.

Why not add some variables to `config.h` to override the `alphas` array with those variables before `setup()` is called?

**Comment by bakkeby** [comment]
That is the idea behind the #275 pull request.

I have an alternative implementation on the experimental [vertigo](https://github.com/bakkeby/dwm-flexipatch/tree/vertigo) branch building on proposal #264. Not sure if any of this is something that I'd bundle into dwm-flexipatch as it is would be taking it in a direction that was not part of the primary goal of the project.

**Reply by ghost** [reply]
I think #275 could be merged in as a separate patch, as it wouldn't be very difficult to do. Considering this project has mandatory patches now such as barmodules the project isn't really identical to a dwm 6.4 anyway.

**Reply by bakkeby** [reply]
Yes but besides adding ~150 lines to that xrdb loading it does also add another maintenance burden; more code to add and maintain if adding a simple colour scheme.

It also assumes the xresource pattern used (specifically thinking of the dot separation), which is arguably better than what is there for the rest of the resources.

I am also thinking that all of this could also be added via config with the [xresources](https://dwm.suckless.org/patches/xresources/) patch.

Being able to control the alpha for the foreground colours I _think_ makes sense for status2d drawing for example, but in my experience this doesn't work when printing text.

Overall I am not convinced that the solution in the PR is the best way to go.
